creator-node/src/snapbackSM.js

```diff
@@ -49,21 +48,27 @@ const SyncType = Object.freeze({
   Pending: User replica set management
 */
 class SnapbackSM {
-  constructor (audiusLibs) {
+  constructor (nodeConfig, audiusLibs) {
```

Comment:
modified constructor to accept `nodeConfig` object to make this more easily testable. see code change in `test/snapbackSM.test.js`